src/components/MakeWidget/subcomponents/RateField/RateField.tsx

```diff
@@ -0,0 +1,29 @@
+import Icon from "../../../Icon/Icon";
+import { Text, Wrapper, RateBox, Button } from "./RateField.styles";
+
+export type RateFieldProps = {
+  TokenPair: string[];
+  Rate: number;
+  onChange: (TokenPair: string[]) => void;
+};
+
+export const RateField: React.FC<RateFieldProps> = ({
+  TokenPair,
+  Rate,
+  onChange,
+}) => {
+  function handleChange() {
+    onChange([TokenPair[1], TokenPair[0]]);
+  }
+
+  return (
+    <Wrapper>
+      <Text>{` 1 ${TokenPair[0]} =`}</Text>
+      <RateBox>{Rate.toFixed(1)}</RateBox>
```

Comment:
The rate could potentially be a long number. So you should limit the amount of characters in the `RateBox`.